front-end/src/app/app.module.ts

```diff
@@ -20,6 +22,7 @@ import { HomeComponent } from './home/home.component';
     TeamComponent,
     AddClubFormComponent,
     HomeComponent,
+    PlayersComponent
```

Comment:
Why does PlayersComponent need to be declared here, but PlayerComponent doesn't?

Reply:
I am trying to work this out.. all the docs imply you need everything declared but it seems to work without PlayerComponent declared, but if I comment out PlayersComponent it gets unhappy 🤔